serverless-tools.gemspec

```diff
@@ -44,5 +44,6 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency "aws-sdk-lambda"
   spec.add_runtime_dependency "aws-sdk-s3"
+  spec.add_runtime_dependency "aws-sdk-ecr"
```

Comment:
You might also want to add docker to the spec `requirements` - it doesn't _do_ anything, but informs the user what is required to run the CLI.

Reply:
Done! ✅